packages/rancher-cis-benchmark/charts/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: v1.0.5
+appVersion: v1.0.6
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes cluster
 name: rancher-cis-benchmark
 version: 1.0.6
```

Comment:
1. This version needs to be added here: https://github.com/rancher/rancher/issues/34589
2. Lets add a min `rancher-version` here, ping me and I can get you more info.

Reply:
Is that something that also needs to go here: https://github.com/rancher/system-charts/pull/534?

Reply:
They both should get added to that issue yes, I can add this time.